overtrack_web/overtrack_web/flask_app.py

```diff
@@ -159,6 +159,11 @@ def context_processor():
     app.register_blueprint(overwatch_discord_blueprint, url_prefix='/overwatch/discord_bot')
 
 
+# ------ MISC ------
+from overtrack_web.views.faq import faq_blueprint
+app.register_blueprint(faq_blueprint, url_prefix='/faq')
+
+
```

Comment:
Adding a blueprint is not necessary for single static pages - just add the route under `SIMPLE INFO PAGES`